cmd/cluster-kube-controller-manager-operator/main.go

```diff
@@ -59,5 +45,7 @@ func NewSSCSCommand(ctx context.Context) *cobra.Command {
 	cmd.AddCommand(certsyncpod.NewCertSyncControllerCommand(operator.CertConfigMaps, operator.CertSecrets))
 	cmd.AddCommand(recoverycontroller.NewCertRecoveryControllerCommand(ctx))
 
+	globalflag.AddGlobalFlags(cmd.PersistentFlags(), cmd.Name(), logs.SkipLoggingConfigurationFlags())
```

Comment:
- non deprecates logging flags are present in global options (`PersistentFlags`) by default (see `cli.Run`). The following command omits them and keeps them only in each command that specifically needs them. It also helps with distinguishing exclusively new and deprecated options (present in global options) and clustering each group together.
- user experience can be also enhanced by adding `"logs"` flagset for the logging configuration (example https://github.com/kubernetes/kubernetes/blob/4df70dfd41601077478289ad358e7c10a4f0d0e1/cmd/kube-controller-manager/app/options/options.go#L276)